cli/lib/info.js

```diff
@@ -5,8 +5,9 @@
  * See the LICENSE file for more information.
  */
 
-const
-	appc = require('node-appc'),
```

Comment:
So I think we can still do per-project overrides, or additional rules on top of the common config. I'd agree that the `"strict": "safe"` rule is probably a good one to turn on while Node 4 is our min supported Node.

Personally, I like having multiple `const/`var`/`let` declarations combined.